### PR TITLE
New Test (Channels : Search) - "Find channel" search returns only 50 results when there are more than 50 channels with similar names

### PR DESCRIPTION
**Pull request by lindy65**
#### Summary

New test to ensure that "Find channel" returns only 50 results if there are more than 50 channels with similar names (e.g. test_channel01, test_channel02, etc)

#### Is the feature in stable branch or under review?
- [x] Feature in stable branch (community)

#### Test client
- [x] Browser
- [x] Desktop App

### CODE REVIEW

**Comment by lindy65** [comment]
@saturninoabril - Can you tell me what the check is that is failing?  I'm not sure what it means...  Thanks!

File: data/test-cases/channels/search/test (channels_search) - Find channel search returns only 50 results.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# (Required) Ensure all values are filled up
```
Comment (saturninoabril) [inline data/test-cases/channels/search/test (channels_search) - Find channel search returns only 50 results.md:1]: You missed the triple dashes on top that is used to define YAML frontmatter (metadata section of the markdown file). This as the main cause of the error in the pipeline.
```suggestion
---
# (Required) Ensure all values are filled up
```

File: data/test-cases/channels/search/test (channels_search) - Find channel search returns only 50 results.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+name: "Channels: Search - Find channel search returns only 50 results when there are more than 50 channels with similar names"
+status: Active
+priority: Normal
+folder: channels/search
```
Comment (saturninoabril) [inline data/test-cases/channels/search/test (channels_search) - Find channel search returns only 50 results.md:5]:
```suggestion
folder: Search
```

File: github.code-workspace [hunk 3]
```diff
@@ -0,0 +1,7 @@
+{
+	"folders": [
+		{
+			"path": ".."
+		}
+	]
+}
```
Comment (saturninoabril) [inline github.code-workspace:7]: Please remove this file.

**Review by saturninoabril — CHANGES_REQUESTED** [review]
Thanks Lindy! Glad to see your first contribution in this repo.

re: validation error, it's due to the malformed markdown file, missing `---` on top to properly detect YAML frontmatter.

Overall, looks good to me except for few requests.

File: github.code-workspace (hunk 3 above)
Comment (lindy65) [reply github.code-workspace:7]: I'm not exactly sure how I created this file 😅 I was messing around in VSCode and saw it was somehow created/attached to this PR but I'm not sure how to remove it?  Can you help explain how I remove it from this PR please? Thanks!

Comment (lindy65) [reply github.code-workspace]: OK, I think I found out how to delete the file...  Let me know if I deleted it correctly :)

**Review by saturninoabril — APPROVED** [review]
Great, thanks Lindy!

**Comment by saturninoabril** [comment]
Test saved - MM-T5424

**Pull request merged**